DEVELOPMENT.md

```diff
@@ -0,0 +1,39 @@
+## Run OpenDID without Docker
+
+To run the OpenDID locally,
+
+1. Run the setup or add your `config.yaml` file with your seed.
+
+```bash
+ENDPOINT=peregrine ./scripts/setup.sh "build hill second flame trigger simple rigid cabbage phrase evolve final eight"
```

Comment:
whose seed is this? May be some note how to bump it from faucet. I think it's better if every dev uses their own peregrine account

Reply:
It's a random seed, I removed it.